pygmt/src/blockm.py

```diff
@@ -147,6 +147,16 @@ def blockmean(data=None, x=None, y=None, z=None, outfile=None, **kwargs):
           is not set.
         - None if ``outfile`` is set (filtered output will be stored in file
           set by ``outfile``).
+
+    Examples
+    --------
```

Comment:
```suggestion
    Example
    -------
```

I think it should just be "Example" as there is only one block of code